### PR TITLE
chore: Update README in importsnapshot

### PR DESCRIPTION
**Pull request by TracyCuiCan**
Add a note about import limitation with Snappy compressed data.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-bigtable-hbase/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

**Review by vermas2012 — COMMENTED** [review]
We should move this Note to the end of the import block. Otherwise looks ok.

File: bigtable-dataflow-parent/bigtable-beam-import/README.md [hunk 1]
```diff
@@ -114,6 +114,12 @@ Exporting HBase snapshots from Bigtable is not supported.
 
 ## Importing to Bigtable
 
+---
+**NOTE**
+
+If your snapshot or sequence files are compressed with Snappy, you'll run into errors importing them as of now. We are working on a solution to resolve this error.
```
Comment (vermas2012) [inline bigtable-dataflow-parent/bigtable-beam-import/README.md:120]: Snappy comressed files are not supported with Import pipelines, we are working to add support for snappy compressed files.

**Review by vermas2012 — APPROVED** [review]
approved with suggestions.

File: bigtable-dataflow-parent/bigtable-beam-import/README.md (hunk 1 above)
Comment (TracyCuiCan) [reply bigtable-dataflow-parent/bigtable-beam-import/README.md]: Thanks, done!

Comment (vermas2012) [reply bigtable-dataflow-parent/bigtable-beam-import/README.md]: I am not sure who needs to approve for yoshi-java? Check with Igor?

**Review by igorbernstein2 — APPROVED** [review]
lgtm

**Pull request merged**